gensim/matutils.py

```diff
@@ -533,7 +533,17 @@ def jaccard(vec1, vec2):
 
 
 def jaccard_set(set1, set2):
-    return 1. - float(len(set1 & set2)) / float(len(set1 | set2))
+    """
+    A distance metric between set representation.
+    Returns 1 minus the intersection divided by union.
+    Returns a value in range <0, 1> where values closer to 0 mean less distance and thus higher similarity.
```

Comment:
`less` => `smaller`.

I see this is a potential gotcha for our users, because other metrics in gensim are similarity (not distance). How about we put this info directly into the method name -- `jaccard_distance` instead of `jaccard_set`?